CMakeLists.txt

```diff
@@ -295,6 +310,12 @@ target_link_libraries(ufs PUBLIC esmf)
 
 if(DATM)
   target_link_libraries(ufs PUBLIC datatm)
+elseif(CDEPS_DATM)
+  list(APPEND _ufs_defs_private CDEPS
+                                FRONT_CDEPS_DATM=atm_comp_nuopc)
+  add_dependencies(ufs datm)
+  include_directories(${CMAKE_CURRENT_BINARY_DIR}/CDEPS/datm)
```

Comment:
Why is this "include_directories" necessary? `datm` target should implicitly set all include directories. If you look at line 290, we just define dependency of `ufs` on `fv3atm`, there's no include_directories(....../fv3atm/....)

Reply:
In CDEPS side each data component has its own include and library files such as libdatm.a, and module file for each dam mode etc. plus some other common library files that are shared along the data components. So, I think we need to have data component specific include files at this point. The docn is also defined in same way.

Reply:
Then CDEPS project should define multiple targets, one for each of the data components, and each of those targets should define it's own include directory and library (.a file). The application can then just specify dependency on one (or more) of those data component targets. The application should not directly depend on common/shared libraries. They should be implicit dependencies. UFS should know nothing about those.